ultrasonic/build.gradle

```diff
@@ -14,10 +14,11 @@ android {
 
         minSdkVersion versions.minSdk
         targetSdkVersion versions.targetSdk
-        resConfigs 'cs', 'de', 'en', 'es', 'fr', 'hu', 'it', 'nl', 'pl', 'pt', 'pt-rBR', 'ru', 'zh-rCN', 'zh-rTW'
```

Comment:
I have removed this because it is annoying to maintain and I guess we want to include all languages which we have files for anyway...